Example/Trustylib-Integration-UIKit/Trustylib-Integration-UIKit.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -5,7 +5,7 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/trustedshops-public/etrusted-ios-trustbadge-library",
       "state" : {
-        "revision" : "3a2861984adb81a1aeac8a8d2c82c18db40ea7b3",
+        "revision" : "eafb1f755bd2097a9fee39161efcb41e92dc8eff",
```

Comment:
question: what is this parameter used for?

Reply:
This must be Swift package manager related parameter which is auto updated by XCode.